app.py

```diff
@@ -90,6 +89,13 @@ def decode_authorization_header(headers):
         raise ValueError('invalid signature format: {0}'.format(err))
 
 
+# TODO: move to authorization.py
+def get_authorization_signature(headers):
```

Comment:
Did not we have such function already?

Reply:
Similar functionality is already implemented in `decode_authorization_header`, but in very coupled way. I suggest to refactor it to authorization.py, but out of this PR scope.

Reply:
Got it.
All endpoints is enough with functionality provided, why aren't we enough in this endpoint?
I dont agree that we need to store Signature in storage, what is argumentation for that?